File: matiasmasso/Mat.NET/DT2/DTOImportTransit.vb
Public Class DTOImportTransit
    Inherits DTOImportacio
    Property YearMonthFras As DTOYearMonth
    Property YearMonthAlbs As DTOYearMonth

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOImportTransit.cs
using System;

namespace DTO
{
    public class DTOImportTransit : DTOImportacio
    {
        public DTOYearMonth YearMonthFras { get; set; }
        public DTOYearMonth YearMonthAlbs { get; set; }

        public DTOImportTransit() : base()
        {
        }

        public DTOImportTransit(Guid oGuid) : base(oGuid)
        {
        }
    }
}
